test/sanity.js

```diff
@@ -81,3 +81,22 @@ transformFile('test/fixtures/test-no-svg-or-react.js', {
     throw new Error('Test failed: React import was present');
   }
 });
+
+function transform(testName, callback) {
+  transformFile(`test/fixtures/test-${testName}.jsx`, {
+    presets: ['airbnb'],
+    plugins: [
+      '../../src/index',
+    ],
+  }, callback);
+}
+
+transform('import', (err1, importResult) => {
+  if (err1) throw err1;
+  transform('require', (err2, requireResult) => {
```

Comment:
so if these two tests don't depend on each other, is there a reason for the `transform` abstraction at all? Could they be written in the same style as the rest of the tests?